### PR TITLE
Added masa to dapps list

### PR DESCRIPTION
**Pull request by im-khem**
> To make a submission to the Dapps Page, please complete the task lists
> below; otherwise remove them.

Your Dapp:

- [x] connects to Valora via WalletConnect V1 or DAppKit;
- [x] must be mobile-first or mobile responsive;
- [x] is in one of the existing categories; and
- [x] includes a link to user documentation.

### CODE REVIEW

File: src/valora-dapp-list.json [hunk 1]
```diff
@@ -403,6 +403,15 @@
       "url": "https://app.jumptask.io",
       "listOnAndroid": true,
       "listOnIos": false
+    },
+    {
+      "name": "Masa Prosperity Passport",
+      "id": "masa",
+      "categoryId": "social",
+      "categories": ["social"],
```
Comment (MuckT) [inline src/valora-dapp-list.json:411]: Should categories include `earn` also?

**Comment by MuckT** [comment]
It looks like there might be some cases where the address is passing undefined in the Dapp. After connecting when attempting to navigate to raregems I see: `https://raregems.io/addr/undefined`. Also I see the following error when refreshing /dashboard after connecting.

```Loading green failed! call revert exception [ See: https://links.ethers.org/v5-errors-CALL_EXCEPTION ] (method="balanceOf(address)", data="0x", errorArgs=null, errorName=null, errorSignature=null, reason=null, code=CALL_EXCEPTION, version=abi/5.7.0)```

File: src/valora-dapp-list.json (hunk 1 above)
Comment (im-khem) [reply src/valora-dapp-list.json:411]: Yes, and nft also. Will correct thanks!

**Comment by im-khem** [comment]
@seb

> It looks like there might be some cases where the address is passing undefined in the Dapp. After connecting when attempting to navigate to raregems I see: `https://raregems.io/addr/undefined`. Also I see the following error when refreshing /dashboard after connecting.
> 
> `Loading green failed! call revert exception [ See: https://links.ethers.org/v5-errors-CALL_EXCEPTION ] (method="balanceOf(address)", data="0x", errorArgs=null, errorName=null, errorSignature=null, reason=null, code=CALL_EXCEPTION, version=abi/5.7.0)`

Thanks for the followup. Regarding those issues:

1. Raregems has lots of bugs. We experienced the same issues a couple of months ago
2. The balanceOf error happens when the provider is connected to an unexpected network. i.e. when you try to connect alfajores to a prod instance.

File: src/valora-dapp-list.json [hunk 2]
```diff
@@ -403,6 +403,15 @@
       "url": "https://app.jumptask.io",
       "listOnAndroid": true,
       "listOnIos": false
+    },
+    {
+      "name": "Masa Prosperity Passport",
+      "id": "masa",
+      "categoryId": "social",
+      "categories": ["social", "earn", "nft"],
```
Comment (MuckT) [inline src/valora-dapp-list.json:411]: ~`nft`~ > `nfts`

**Comment by MuckT** [comment]
@im-kbem - I am able to see my nfts on raregems.io with the following url: https://raregems.io/addr/0xDE33e71fAECdEad20e6A8af8f362d2236CbA005f

**Comment by teslashibe** [comment]
@MuckT this shouldn't block us for launch. It's unclear where you are seeing the console log - i.e. RareGems or CPP- some logs are not issues.

Is this a blocker?

**Comment by MuckT** [comment]
@im-khem @teslashibe I've connected via mainnet so it should recognize the network. The issue with rare gems is because variable that is expected to be defined is not.

**Comment by im-khem** [comment]
@MuckT apologies. This issue should be resolved if you give it another check

**Review by MuckT — APPROVED** [review]

**Pull request merged**